### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Mantserov**
[DEMO LINK](https://Mantserov.github.io/react_add-todo-form/)

### CODE REVIEW

File: src/types/types.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  name: string,
+  username: string,
+  email: string,
+}
+
+export interface TodoWithUser {
+  id: number,
+  title: string,
+  completed: boolean,
+  // userId: number,
```
Comment (Anton-Kuchmasov) [inline src/types/types.ts:19]: don't leave comments in code

**Review by Anton-Kuchmasov — APPROVED** [review]
nice! 
but pay more attention to WHAT you're commiting,

in that PR you want to `merge 10 commits into [mate-academy:develop](https://github.com/mate-academy/react_add-todo-form/tree/develop) from [Mantserov:develop](https://github.com/Mantserov/react_add-todo-form/tree/develop)`

it's not okay

**Comment by Mantserov** [comment]
> nice! but pay more attention to WHAT you're commiting,
> 
> in that PR you want to `merge 10 commits into [mate-academy:develop](https://github.com/mate-academy/react_add-todo-form/tree/develop) from [Mantserov:develop](https://github.com/Mantserov/react_add-todo-form/tree/develop)`
> 
> it's not okay

**Comment by Mantserov** [comment]
> nice! but pay more attention to WHAT you're commiting,
> 
> in that PR you want to `merge 10 commits into [mate-academy:develop](https://github.com/mate-academy/react_add-todo-form/tree/develop) from [Mantserov:develop](https://github.com/Mantserov/react_add-todo-form/tree/develop)`
> 
> it's not okay

To be honest I saw this stuff, but I thought it's how it should be, I dk what error led to this :/ And I'm not sure how to fix it now!

**Comment by BudnikOleksii** [comment]
> > nice! but pay more attention to WHAT you're commiting,
> > in that PR you want to `merge 10 commits into [mate-academy:develop](https://github.com/mate-academy/react_add-todo-form/tree/develop) from [Mantserov:develop](https://github.com/Mantserov/react_add-todo-form/tree/develop)`
> > it's not okay
> 
> To be honest I saw this stuff, but I thought it's how it should be, I dk what error led to this :/ And I'm not sure how to fix it now!

You created PR to the wrong branch, that why

**Pull request closed without merging**